wordpress-mobile/WordPress-iOS
# Enable the milestoneNotifications feature flag

### To test:
1. Sandbox public-api.wordpress.com and s.wp.com
2. Launch the app
3. Tap on Notifications
4. Locate a Milestone notification
5. Tap on it
6. Verify the styling is correct
6. Make sure the confetti works in both light and dark mode
7. Tap on the Take me there button and verify you're brought to the correct insights view

### PR submission checklist:
